portfolio/src/main/webapp/index.html

```diff
@@ -75,16 +75,15 @@ <h2>Who am I?</h2>
 			</section>
 			<section>
 				<form action="/data" method="POST">
-					<label for="name">Name:</label>
+					<input type="text" id="name" name="name" placeholder="Name...">
 					<br>
-					<input type="text" id="name" name="name">
 					<br>
-					<label for="content">Comment:</label>
-					<br>
-					<input type="text" id="content" name="content">
-					<input type="submit" value="Submit">
+					<textarea type="text" id="content" name="content" placeholder="Your Comment..."></textarea>
```

Comment:
Unlike `input` elements, I don't think `textarea` elements attach any meaning to the `type` attribute; they are already expecting text.